data/administrations/cnil.yml

```diff
@@ -0,0 +1,15 @@
+slug: cnil
+short: CNIL
+long: Commission Nationale de l'Informatique et des Libertés (CNIL)
+site: https://www.cnil.fr/
+contact: https://www.cnil.fr/contact
+logoType: portrait
+dataSources:
+  - label: Liste des organismes ayant désigné un E-Délégué à la protection des données (DPO)
+    apiSlug: liste-dpo
```

Comment:
Then we need to add a new API in liste-dpo in `/data/administrations/dinum.yml` and create the monitor in https://updown.io/